src/main/webapp/js/basicview/basicview.soy

```diff
@@ -26,15 +26,13 @@
       <link rel="stylesheet"
           href="https://code.getmdl.io/1.3.0/material.indigo-red.min.css">
       <link rel="stylesheet" href="../../css/table.css">
+      <link rel="stylesheet" href="../../css/page.css">
```

Comment:
This works? The path should be relative to the webapp directory because webapp is the document root.

Reply:
It works.